android/src/main/java/com/apptentive/android/sdk/reactlibrary/ApptentiveModule.kt

```diff
@@ -237,41 +281,78 @@ class ApptentiveModule(reactContext: ReactApplicationContext) : ReactContextBase
     // Key/Sig
     val apptentiveKey = credentials.getString("apptentiveKey")!!
     val apptentiveSignature = credentials.getString("apptentiveSignature")!!
+
     // Create configuration instance
     val apptentiveConfiguration = ApptentiveConfiguration(apptentiveKey, apptentiveSignature)
+
     // Set flags, or defaults if not present
-    apptentiveConfiguration.logLevel = parseLogLevel(if (credentials.hasKey("logLevel")) credentials.getString("logLevel") ?: "info" else "info")
-    apptentiveConfiguration.setShouldEncryptStorage(if (credentials.hasKey("shouldEncryptStorage")) credentials.getBoolean("shouldEncryptStorage") else false)
-    apptentiveConfiguration.setShouldSanitizeLogMessages(if (credentials.hasKey("shouldSanitizeLogMessages")) credentials.getBoolean("shouldSanitizeLogMessages") else true)
-    apptentiveConfiguration.isTroubleshootingModeEnabled = if (credentials.hasKey("troubleshootingModeEnabled")) credentials.getBoolean("troubleshootingModeEnabled") else true
-    // Return configuration
-    return apptentiveConfiguration
-  }
+    if (credentials.hasKey("logLevel")) {
+      apptentiveConfiguration.logLevel =
+        parseLogLevel(credentials.getString("logLevel") ?: "info")
+    }
 
-  // Represents the name of the module
-  override fun getName(): String { return "ApptentiveModule" }
+    // Set distribution name and version
+    if (credentials.hasKey("distributionName")) {
+      apptentiveConfiguration.distributionName = credentials.getString("distributionName") ?: "React Native"
+    }
 
-  private fun getApplicationContext(): Application {
-    return mReactContext.currentActivity?.applicationContext as Application
+    if (credentials.hasKey("distributionVersion")) {
+      apptentiveConfiguration.distributionVersion = credentials.getString("distributionVersion").orEmpty()
```

Comment:
Did you verify if the `distributionVersion` and `distributionName` were passed correctly?

Reply:
Yep